pkgs/applications/science/biology/seaview/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, pkgs }:
```

Comment:
Please rely on the dependencies (like fltk and libjpeg) directly here, and get rid of the pkgs argument

Reply:
Thanks for the comment.